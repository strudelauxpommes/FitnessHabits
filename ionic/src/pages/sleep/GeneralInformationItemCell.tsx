import { IonLabel, IonRow, IonCol } from '@ionic/react';
import React from 'react';
export function GeneralInformationItemCell(props: {
  labelTitle: React.ReactNode;
  labelSubtitle: React.ReactNode;
  labelValue: React.ReactNode;
}) {
  return <IonRow>
    <IonCol size='9'>
      <IonRow>
        <IonLabel>
          {props.labelTitle}
        </IonLabel>
      </IonRow>
      <IonRow>
        <IonLabel>
          {props.labelSubtitle}
        </IonLabel>
      </IonRow>
    </IonCol>
    <IonCol size='3'>
      <IonLabel>
        {props.labelValue} heure(s)
      </IonLabel>
    </IonCol>
  </IonRow>;
}
