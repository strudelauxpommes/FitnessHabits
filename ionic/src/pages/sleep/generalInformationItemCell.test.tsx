import React from 'react';
import {GeneralInformationItemCell} from './GeneralInformationItemCell';
// import SleepComponent from '../../pages/components/SleepComponent'
import renderer from 'react-test-renderer'

// import Adapter from 'enzyme-adapter-react-16';
// import {configure, mount, shallow} from 'enzyme'

// configure({ adapter: new Adapter() });

test('renders correctly', () => {


    const tree = renderer.create(<GeneralInformationItemCell labelTitle='Dernière période' labelSubtitle='31/10/2019' labelValue='7'  />).toJSON();
    expect(tree).toMatchSnapshot();
    // expect(tree).toHaveProperty('ion-label', 'Intro')
});