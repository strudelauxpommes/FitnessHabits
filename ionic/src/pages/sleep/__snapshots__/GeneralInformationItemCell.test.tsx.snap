// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ion-row>
  <ion-col>
    <ion-row>
      <ion-label>
        Dernière période
      </ion-label>
    </ion-row>
    <ion-row>
      <ion-label>
        31/10/2019
      </ion-label>
    </ion-row>
  </ion-col>
  <ion-col>
    <ion-label>
      7
       heure(s)
    </ion-label>
  </ion-col>
</ion-row>
`;
